about/pricing.md

```diff
@@ -0,0 +1,72 @@
+We'll fill in more details in this section as we understand them better, but in the meantime here are a few guidelines that we'll use for our pricing:
+
+2i2c DevOps fees should be:
+
+- Competitive with other "Data Science environment as a service" offerings.
+- An amount that is sustainable for the communities we serve, with mechanisms to accommodate institutions with fewer resources.
+- An amount that allows 2i2c to both sustain itself and thrive, and covers our time developing and supporting open source projects as part of the service.
+
+## Cloud costs
+
+### How much will my cloud costs be?
+
+This is difficult to estimate ahead of time, because it is **heavily** dependent on the primary use-case and size of your community, and because cloud vendors change the cost of their machines often (though they don't change by much).
+
+To get a rough estimate of your cloud costs, here is a rough guide that is based on **Memory (RAM)**. It is a reasonable rule of thumb as long as you don't any particularly complex needs like extremely large datasets or scalable computing/GPUs.
+
+1. **Read [the Zero to JupyterHub cost projection documentation](z2jh:cost)**. This is a nice high-level overview of the factors that drive the cost of JupyterHub deployments on commercial cloud.
+2. **Estimate memory available to each user**. The amount of RAM needed for each user is often the biggest driver of cloud cost. Decide the "maximum" amount of RAM that a user will generally need, and multiply that by 1.5x.
+3. **Determine how many average simultaneous users you'd like a hub to support**. This isn't necessarily the total size of your community, but how many people you think will be using the hub *at the same time*.
+4. **Look up the monthly price of an `e2-highmem4` machine**. This is a basic machine that is suitable for many use-cases. [Go to the Google Cloud pricing page](https://cloud.google.com/compute/vm-instance-pricing) and go to the `E2 high-memory machine types` section. In that section, look for the `e2-highmem-4` monthly price.
+
+Now calculate a rough estimate of your monthly cloud cost with the following formula:
+
+```
+(n_avg_simultaneous_users * memory_per_user) / 16GB * monthly_price
+```
+
+So for example, if you have a community of **100 total users** but expect an average of only 20 of them to be active at the same time, and each user requires **4GB** of RAM, then with a monthly `e2-highmem4` price of $97.83 (as of {sub-ref}`today`), then your monthly cost will be around:
+
+```
+(20 * 4) / 16 * 97.83 = $489.15 / month or $5,869.80 / year
+```
+
+Or, around **$4.80 per user per month** in cloud costs. If you required **2GB RAM** instead of **4GB RAM**, cloud costs would likely drop to around **$2.40 per user per month**.
+
+:::{warning}
+This is just a rough estimate! As mentioned above, the actual costs will vary from this, but this amount should be correct within an order of magnitude.
+:::
```

Comment:
OK, all this section is heavily biased towards the GCP stuff... I understand we should start with something but can we add, at least, a warning about estimates differences by cloud providers?

Reply:
Good point - i was just using it as a single estimate to avoid overloading people with information. I'll add a note about comparable machines for the other providers (and the z2jh link has a lot of good resources too)

Reply:
I've added a note about this at the beginning of the section